doc/example/reference.cpp

```diff
@@ -490,14 +497,17 @@ namespace invocation_example_4
 //[ invocation_example_4
 MOCK_CLASS( mock_class )
 {
-    MOCK_STATIC_METHOD( method, 1, void( int ) )
+    MOCK_STATIC_METHOD( method1, 1, void( int ) )
+    MOCK_STATIC_METHOD( method2, 1, void( int ) )
 };
 
 BOOST_AUTO_TEST_CASE( demonstrates_setting_up_an_invocation_on_a_mock_static_method )
 {
     mock_class c;
-    MOCK_EXPECT( c.method ).once();
-    MOCK_EXPECT( mock_class::method ).once(); // does the same
+    MOCK_EXPECT( c.method1 ).once();
+    MOCK_EXPECT( mock_class::method2 ).once(); // does the same
```

Comment:
Well, it doesn’t do the same anymore if they’re on different methods.

Reply:
Adapted the comment, better?

Reply:
OK.